### ISSUE TITLE
Using formatNumber in controller causes error

### ISSUE THREAD

**Issue by terryroe** [yahoo/ember-intl#48]
```
ember -v
version: 0.1.15
node: 0.10.35
npm: 2.1.8
```

Console Output
```
$E.intl.formatNumber(1000)
intl.js:132 Uncaught TypeError: Cannot read property 'locales' of undefined 
intl.js:132 exports.default.ServiceKlass.extend._format
intl.js:123 exports.default.ServiceKlass.extend.formatNumber
VM6635:2 (anonymous function)
VM6491:777 InjectedScript._evaluateOn
VM6491:710 InjectedScript._evaluateAndWrap
VM6491:626 InjectedScript.evaluate
```

I used the Application Controller for $E above.  Not sure if use of this function is supported in controllers, but it looks to me like it is.

**Comment by jasonmit** [comment]
What version of ember?  Do you have a branch anywhere with this issue?  Any other formatters not working?Will investigate once I get back to my desk.Sent from Yahoo Mail for iPhoneAt Feb 19, 2015, 12:28:09 PM, Terry Roe wrote:ember -v
version: 0.1.15
node: 0.10.35
npm: 2.1.8


Console Output

$E.intl.formatNumber(1000)
intl.js:132 Uncaught TypeError: Cannot read property 'locales' of undefined 
intl.js:132 exports.default.ServiceKlass.extend._format
intl.js:123 exports.default.ServiceKlass.extend.formatNumber
VM6635:2 (anonymous function)
VM6491:777 InjectedScript._evaluateOn
VM6491:710 InjectedScript._evaluateAndWrap
VM6491:626 InjectedScript.evaluate


I used the Application Controller for $E above.  Not sure if use of this function is supported in controllers, but it looks to me like it is.

**Comment by terryroe** [comment]
Sorry Jason.  I should have thought to include the ember version.  It's 1.10.0.  I'm sorry, I don't have a public branch with this.  All of the other formatters produce the same error.  They are all on a line like this:  
```
 var locales = makeArray(options.locales);
```

**Comment by jasonmit** [comment]
I fixed the problem, just need to run out for a minute to a talk then want to write a test to prevent regression.  Thanks for reporting this, I haven't documented invoking the intl format functions directly, outside of the helpers, perhaps I will now :)

**Comment by jasonmit** [comment]
Published 1.1.1 which contains a fix for this
